### PR TITLE
add aravind portfolio

### PR DESCRIPTION
**Pull request by Evavic44**
### Portfolio Author

@aravindballa

### Portfolio Link?
https://aravindballa.com/

### GitHub Link:
https://github.com/aravindballa/website

### Tech Stack
- NextJS
- TypeScript
- Tailwind CSS
- Vercel

Testing auto-deploy action
Reference: #225

### CODE REVIEW

**Comment by Evavic44** [comment]
Hey, @vimode I think I know what the problem is. I noticed the action only runs on the branch specified in the `yml` workflow file, and since we specified `main` branch, the preview will point to the main branch and not an external branch like `add-ahoy` for example.

What this means is that until the PR is merged or done in main, the action will not run. The screenshot below illustrates this properly

deploy action

Also, this explains why Vercel preview worked on the `[workflow/autodeploy]` branch you created when the workflow was added.


## In summary
Updating the `portfolio.md` only happens when the PR is merged which I think is still okay. 🙂

**Comment by vimode** [comment]
Hey @Evavic44 Alright good analysis. I totally forgot I had the custom branch name just for testing.

Although I have to admit, I did not think that keeping it for just for `main` would only trigger it when **PR is merged**, I was thinking all PRs are requesting to merge with `main` might be considered a push request to `main` but now that you explained it, it makes sense that a merge request is not as same as actually pushing to `main`. 

Quick question out of curiosity, if a PR request is made in the `main` directly, without creating a new branch(to merge with main) the action should run and generate preview in the PR, isn't it?

> Updating the portfolio.md only happens when the PR is merged which I think is still okay. 🙂

Yes thats wise, keeping it for all branches will run the action twice unnecessarily and mostly likely after merge will fail as its already done the task in the first run. 

So, no change in the workflow necessary. 👍

**Comment by Evavic44** [comment]
> Quick question out of curiosity, if a PR request is made in the main directly, without creating a new branch(to merge with main) the action should run and generate preview in the PR, isn't it?

Yes it will.

**Comment by aravindballa** [comment]
Thanks for adding mine @Evavic44 😄

Great work on this repo!

**Comment by Evavic44** [comment]
> Thanks for adding mine @Evavic44 😄
> 
> Great work on this repo!

You're welcome, @aravindballa Cool portfolio by the way. 👍🏽
